code/kg2/build-kg2.sh

```diff
@@ -1,326 +1,336 @@
+           ${TEST_ARG} \
+           ${UNICHEM_OUTPUT_TSV_FILE} \
+           ${UNICHEM_OUTPUT_FILE}
+
+echo "running ncbigene_tsv_to_kg_json.py"
+
+## Make JSON file for NCBI Gene
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/ncbigene_tsv_to_kg_json.py \
+           ${TEST_ARG} \
+           ${NCBI_GENE_TSV_FILE} \
+           ${NCBI_GENE_OUTPUT_FILE}
+
+echo "running dgidb_tsv_to_kg_json.py"
+
+## Make JSON file for DGIDB
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/dgidb_tsv_to_kg_json.py \
+           ${TEST_ARG} \
+           ${DGIDB_DIR}/interactions.tsv \
+           ${DGIDB_OUTPUT_FILE} 2> ${DGIDB_DIR}/dgidb-tsv-to-kg-json.log
+
+echo "running repodb_csv_to_kg_json.py"
+
+## Make JSON file for REPODB
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/repodb_csv_to_kg_json.py \
+           ${TEST_ARG} \
+           ${REPODB_INPUT_FILE} \
+           ${REPODB_OUTPUT_FILE} 2> ${REPODB_DIR}/repodb-csv-to-kg-json.log
+
+echo "copying RTX Configuration JSON file from S3"
+
+${S3_CP_CMD} s3://${S3_BUCKET}/${RTX_CONFIG_FILE} ${BUILD_DIR}/${RTX_CONFIG_FILE}
+
+echo "extracting KG JSON representation of RTX KG1, from the Neo4j endpoint"
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/rtx_kg1_neo4j_to_kg_json.py \
+           ${TEST_ARG} \
+           --configFile ${BUILD_DIR}/${RTX_CONFIG_FILE} \
+           ${KG1_OUTPUT_FILE}
+
+echo "running merge_graphs.py"
+
+## Merge all the KG JSON files
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/merge_graphs.py \
+           ${TEST_ARG} \
+           --kgFiles ${OUTPUT_FILE_FULL} \
+                     ${SEMMED_OUTPUT_FILE} \
+                     ${UNIPROTKB_OUTPUT_FILE} \
+                     ${ENSEMBL_OUTPUT_FILE} \
+                     ${UNICHEM_OUTPUT_FILE} \
+                     ${CHEMBL_OUTPUT_FILE} \
+                     ${NCBI_GENE_OUTPUT_FILE} \
+                     ${DGIDB_OUTPUT_FILE} \
+                     ${REPODB_OUTPUT_FILE} \
+                     ${KG1_OUTPUT_FILE} \
+           --kgFileOrphanEdges ${OUTPUT_FILE_ORPHAN_EDGES} \
+           ${FINAL_OUTPUT_FILE_FULL}
+
+echo "get_nodes_json_from_kg_json.py"
+
+## Get a JSON file with just the nodes in it
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/get_nodes_json_from_kg_json.py \
+           ${FINAL_OUTPUT_FILE_FULL} \
+           ${OUTPUT_NODES_FILE_FULL}
+
+echo "report_stats_on_json_kg.py (full KG)"
+
+## Generate a JSON report of statistics on the KG
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/report_stats_on_json_kg.py \
+           ${FINAL_OUTPUT_FILE_FULL} \
+           ${REPORT_FILE_FULL}
+
+echo "filter the JSON KG and remap predicates"
+
+## Filter the JSON KG and remap predicates:
+
+${VENV_DIR}/bin/python3 -u ${CODE_DIR}/filter_kg_and_remap_predicates.py \
+           --dropNegated \
+           ${PREDICATE_MAPPING_FILE} \
+           ${CURIES_TO_URLS_FILE} \
+           ${FINAL_OUTPUT_FILE_FULL} \
+           ${SIMPLIFIED_OUTPUT_FILE_FULL}
+
+echo "generating slimmed-down kg2 (issue #597)"

```

Comment:
Added this block of code to run the slimming script on the completed but uncompressed kg2 file